starter-code/css/main.css

```diff
@@ -7,4 +7,81 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+
+}
+
+.container{
+  display: flex;
+  justify-content: center;
+  padding-top: 110px;
```

Comment:
It works here, but definitely not the cleanest way of centering your content ;)
What you could do instead is:
```css
container-element {
  display: flex;
  justify-content: center;
  align-items: center;
} 

child-element {
  width: 70%; /*This is an example, you can give it whatever width, as long as it's defined*/
  height: 200px; /*This is an example, you can give it whatever width, as long as it's defined*/
}
```